scripts/testnet-deploy.js

```diff
@@ -22,22 +20,18 @@ const getClient = async () => {
       instance: await Node(config)
     }],
     accounts: [MemoryAccount({keypair: keypair})],
-    networkId: 'ae_uat',
     compilerUrl: config.compilerUrl
   });
 };
 
 const deploy = async () => {
   const client = await getClient();
-  const oracleContractAddress = ""
-
-  const contractV1 = await client.getContractInstance(TIPPING_V1_CONTRACT);
-  await contractV1.methods.init(oracleContractAddress, keypair.publicKey);
-  console.log("v1", contractV1.deployInfo.address);
 
+  await client.spend(1, "ak_2VnwoJPQgrXvreUx2L9BVvd9BidWwpu1ASKK1AMre21soEgpRT");
   const contractV2 = await client.getContractInstance(TIPPING_V2_CONTRACT);
-  await contractV2.methods.init(oracleContractAddress);
+  await contractV2.methods.init("ct_2PbBVGroJ2G618tHbWEWRrrxxf1BAn7Qt7BoNcLh3LqsSck8Lu");
```

Comment:
should it be like this?
```suggestion
  await contractV2.init("ct_2PbBVGroJ2G618tHbWEWRrrxxf1BAn7Qt7BoNcLh3LqsSck8Lu");
```

Reply:
didn't upgrade this file it seems, will check monday